app/google-services.json

```diff
@@ -0,0 +1,86 @@
+{
+  "project_info": {
+    "project_number": "985111569241",
+    "firebase_url": "https://irish-rail-app-2017.firebaseio.com",
+    "project_id": "irish-rail-app-2017",
+    "storage_bucket": "irish-rail-app-2017.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:985111569241:android:9876c226c5b5e9a0",
+        "android_client_info": {
+          "package_name": "com.codeforireland.vacanthome"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "985111569241-qongkl4jh20g3fevahd4mgci31gl7i36.apps.googleusercontent.com",
+          "client_type": 1,
+          "android_info": {
+            "package_name": "com.codeforireland.vacanthome",
+            "certificate_hash": "c9545922b24855f2f8a92d3aacfcb6a1003db4b0"
+          }
+        },
+        {
+          "client_id": "985111569241-pfermgl345ie1nmbfuj31u38b86a8ibp.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyCBb3YYi57eHY-u7CH8dTBoMa8jjxE84YU"
```

Comment:
As I mentioned in person today, I'm not sure if we should be committing active API keys here

Reply:
Done